### DISCUSSION TITLE
how to prevent from a request continue to remote server when i make a fake response

### DISCUSSION THREAD

**Discussion by fr0zenrain** [mitmproxy/mitmproxy#7388] [Q&A] [answered]
i want to make a cache server,so i need override the request and response. in my request i will make a response with **http.Response.make**.I need improve the performance,but how do i prevent from a request continue to server,because i had made a fake response.

**Comment by Prinzhorn** [comment, marked as answer]
https://github.com/mitmproxy/mitmproxy/blob/main/examples/addons/http-reply-from-proxy.py

`+`

`--set connection_strategy=lazy` to prevent server unused connection.

**Reply by fr0zenrain** [reply]
i missed this option,thanks for your help.
